### ISSUE TITLE
why zuul2 samples dont have application.yml

### ISSUE THREAD

**Issue by userAshish** [Netflix/zuul#519]
When I try to start zuul2 sample I am always 
Error: Your request of "passthrough" was not found on redirect platform.

**Comment by userAshish** [comment]
or any configuration file to define and map paths, I am coming from Zuul1 spring boot world , so this appeared to be way diffrent than what we have in Zuul1 , do we have any migration guide from Zuul1 to zuul2?

**Comment by tmack8001** [comment]
tldr; Migrating from spring-cloud-netflix -> zuul/2.x isn't just a swap replacement.

@userAshish spring-boot world had a while set of configurations that would read your application.yml and use zuul/1.x library internally to setup routing with.

In zuul/2.x which isn't spring based (and zuul/1.x wasn't either) and has been called out in spring-boot community not being ported back to spring-cloud-netflix projects (which you are using). This means you would need to take on the static routing rules which you have setup in your previous application.yml and port these. This logic is essentially within a few of the Zuul Filters that are setup in spring-cloud-netflix that you are using today PreDecorationFilter (selecting the ribbon client to proxy to) and SimpleRouteFilter (doing the routing logic itself).

**Comment by userAshish** [comment]
Thanks @tmack8001  , to port all my entries from application.yml like 
 userpath:
      path: /user/**
      url: https://myuser.com
      stripPrefix: false
....

I dont see any example like this for zuul2, I am currently trying to understand https://github.com/Netflix/zuul/tree/2.1/zuul-sample .
can you please guide me to any good example that can help me understand replacement of application.yml entries.

**Comment by dashprateek** [comment]
@userAshish  check this out https://github.com/dashprateek/zuul2-sample

**Comment by artgon** [comment]
As mentioned, the `application.yml` is a spring-cloud construct. However the solution that @dashprateek shows above is a good way to get around it.
